test/docs/testplans/vnet.md

```diff
@@ -0,0 +1,156 @@
+# Table of content
+
+1. [Objectives](#objectives)
+2. [Requirements](#requirements)
+3. [Automation](#automation)
+4. [Test Suites](#test-suites)
+    - [Outbound VNET routing](#outbound-vnet-routing)
+    - [Inbound VNET routing](#inbound-vnet-routing)
+    - [Integration](#integration)
+    - [Negative](#negative)
+    - [Scaling & Performance](#scaling--performance)
+    - [To clarify / Future](#to-clarify--future)
+
+---
+
+# Objectives
+
+The VNET-to-VNET scenario is the starting point to design, implement and test the core DASH mechanisms in VM to VM communication in VNET, using an Appliance for rules and routing offload.
+
+The scenario allows the following:
+- Route/LPM support
+- Underlay IPv4 and IPv6
+- Stateful ACL support
+- TCP state tracking on flows
+- Telemetry and Monitoring
+
+## Requirements
+
+### Scale
+| Item                   |    Expected value    |
+|------------------------|:--------------------:|
+| VNETs                  |         1024         |
+| ENI per card           |          64          |
+| Routes per ENI         |         100k         |
+| NSGs per ENI           |     5 in + 5 out     |
+| ACLs per ENI           |   10x100K prefixes   |
+| ACLs per ENI           | 10x10K SRC/DST ports |
+| CA-PA Mappings per ENI |         160k         |
+| Active Connections/ENI |  1M (Bidirectional)  |
+
+### Performance
+| Item                | Expected value |
+|---------------------|:--------------:|
+| CPS per 200G        |      3.75M     |
+| Flows per ENI       |         1M     |
+| Flows per 200G card |        64M     |
+| Flows per 400G card |       128M     |
+
+
+### Other
+
+1. Bulk update of LPM and CA-PA Mapping tables.
+2. Mapping updates can occur as much as 100 mappings/sec
+3. ACL operations (rules adding/deleting) per group for a stage must be handled atomically.
+4. Support ability to get all ACL rules/groups based on guid.
+5. During VNET or ENI delete, implementation must support ability to delete all mappings or routes in a single API call.
+6. Add and Delete APIs are idempotent.
+7. During a delete operation, if there is a dependency, implementation shall return error and shall not perform any force-deletions or delete dependencies implicitly.
+8. During a bulk operation, if any part/subset of API fails, implementation shall return error for the entire API.
+9. Implementation must have flexible memory allocation for ENI and not reserve max scale during initial create (e.g 100k routes). This is to allow oversubscription.
+10. Implementation must not have silent failures for APIs.
+
+More details may be found in [DASH SONiC HLD](https://github.com/Azure/DASH/blob/main/documentation/general/design/dash-sonic-hld.md#15-design-considerations).
```

Comment:
Link is broken, `design` was removed from path quite some time ago.
Also, change `Azure` to `sonic-net`